app/build.gradle

```diff
@@ -78,19 +77,21 @@ dependencies {
 
     testCompile 'junit:junit:4.12'
 
-    //App build types
+    // App build types
     packagedCompile 'com.leanplum:Leanplum:' + LP_VERSION
```

Comment:
could you please change to 'com.leanplum:leanplum-gcm:' and add 'com.leanplum:leanplum-location:'